src/css/_saved-spaces-list.scss

```diff
@@ -57,6 +57,8 @@
     height: 25px;
     margin: 0 2px;
     width: 95%;
+    padding-left: 22px;
```

Comment:
I meant for this to be in the spaces PR, I think while I was doing some rebasing it may have gotten dropped

Reply:
Right on.